etcdserver/config.go

```diff
@@ -59,6 +59,9 @@ type ServerConfig struct {
 
 	// ClientCertAuthEnabled is true when cert has been signed by the client CA.
 	ClientCertAuthEnabled bool
+
+	TokenType string
```

Comment:
Or maybe only have `TokenType` instead of `TokenOpts` and have the flag be `--auth-token-type` since this string completely encodes the token type? /cc @xiang90